### PR TITLE
Meta: Add `npm run new` to add new features

### PR DESCRIPTION
**Pull request by fregante**
```
> nr new -- one-click-issue-closure
✨ Feature created! ✨
source/features/one-click-issue-closure.tsx

After you're done, also add the feature to ./readme.md; Don't worry about the screenshot yet

Refer to https://github.com/refined-github/refined-github/blob/main/contributing.md for more information.
```

### CODE REVIEW

**Comment by 134130** [comment]
Suggesting How about automatically or guiding "append import line to refined-github.ts"

File: build/new-feature.sh [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+void features.add(import.meta.url, {
+	include: [
+		pageDetect.isPR, // Find the one you need on https://refined-github.github.io/github-url-detection/
+	],
+	init,
+});
+" >> "source/features/$1.tsx"
+
+echo "import './features/$1.js';" >> ./source/refined-github.ts
```
Comment (fregante) [inline build/new-feature.sh:31]: It's there

**Comment by yakov116** [comment]
This is awesome 👌

**Comment by kidonng** [comment]
The script seems simple enough to be written in JS so it also runs on Windows (well I kinda get that if you are a developer you would better have shell available through WSL/Cygwin/whatever, but still).

**Comment by fregante** [comment]
AI can probably take care of the conversion. Feel free to send a PR

**Pull request merged**